### DISCUSSION TITLE
Sollwertvorgabe ändert sich trotz hoher Hysteresevorgabe

### DISCUSSION THREAD

**Discussion by Napizaki** [hoylabs/OpenDTU-OnBattery#149] [Show and tell]
Hallo und danke erstmal für dieses wunderbare Projekt!

Ich habe in der aktuellen Version festgestellt, dass der Inverter, trotz meiner Meinung nach, hohen Hysterese, jedes Mal einen neuen Sollwert bekommt. Habe ich da was falsch eingestellt, oder liegt dort tatsächlich ein Fehler vor? Siehe Bilder im Anhang

Eine Frage zu der Hysterese hätte ich noch: wenn ich 50W eingebe, sind das dann +/- 50W vom Sollwert oder +/- 25W?

Vielen Dank im Voraus!

![image](https://user-images.githubusercontent.com/126354867/228498608-b2a20c29-9ca2-4e2d-966f-ef763da9e04c.png)
![image](https://user-images.githubusercontent.com/126354867/228498693-db8473bf-23a3-40fa-9716-b9f6b98a9e0e.png)

**Comment by helgeerbe** [comment]
Hi. Bin leider unterwegs und kann erst nächste Woche in den Code schauen. 

Die Hystereses greift nur bei abgeschalteten SolarPassThrough. Dann sollte  bei einem Sollwert +- Hystereses nicht geregelt werden. Bei 50 W Sollwert und 10W Hystereses also keine Regelung zwischen 40 und 60 W.

**Reply by Napizaki** [reply]
Danke für die schnelle Antwort.
SolarPassThrough ist abgeschaltet. Sollwert ist bei 25W, Hysterese ist bei 50W. Er sollte also laut meinem Verständnis nicht nachregeln bei den im Diagramm gezeigten Schwankungen. Es wird aber jedes Mal nachgeregelt.

Keine Eile - alles Andere funktioniert ja gut.

Eine Andere Frage: Ich habe den Akku an beiden Kanälen meines HM-600 angeschlossen. Dieses kann ich so nicht in der Software konfigurieren - ich habe dann nur Kanal 0 eingestellt. Kann es da zu Problemen kommen? Wäre es vorstellbar, dass man statt des Drop-Down Menüs, Checkboxen einbaut - da wäre man flexibler bei der Auswahl.

**Reply by Adminius** [reply]
das ist mir auch schon aufgefallen. Ich habe ja auch HM600 an beiden Eingängen hängt der Akku. Früher war das Limit für kompletten WR gedacht...

**Reply by helgeerbe** [reply]
Ist immer noch so. Ich brauche den Kanal wo ac power und dc gesamt volt angegeben ist. Kanal sollte eigentlich immer 0 sein.
